protocol/morse/protocol.go

```diff
@@ -132,7 +132,7 @@ func (p *Protocol) AvailableEndpoints(
 // BuildRequestContextForEndpoint builds a new request context for a given service ID and endpoint address.
 // Implements the gateway.Protocol interface.
 func (p *Protocol) BuildRequestContextForEndpoint(
-	_ context.Context,
+	ctx context.Context,
```

Comment:
NIT: ctx is not used in the method. Using `_` as name makes the code more readable by clarifying this.